Orange/preprocess/score.py

```diff
@@ -196,3 +203,70 @@ class Gini(ClassificationScorer):
     """
     def from_contingency(self, cont, nan_adjustment):
         return (_gini(np.sum(cont, axis=1)) - _gini(cont)) * nan_adjustment
+
+
+class ReliefF(Scorer):
+    feature_type = Variable
+    class_type = DiscreteVariable
+
+    def __init__(self, n_iterations=None, k_nearest=None):
+        self.n_iterations = n_iterations
+        self.k_nearest = k_nearest
+
+    def score_data(self, data, feature):
+        assert len(data.domain.class_vars) == 1, 'TODO: multi-class?'
+        assert data.domain.class_var.is_discrete
```

Comment:
Sure it. I left them in because those conditions weren't set in stone. If there are multiple classes, ReliefF could score the features according to all of them (returning a 2D array), or to the first one, ... I wasn't sure what the best interface was. Did fix it now, though.